lib/samson/secrets/key_resolver.rb

```diff
@@ -9,6 +9,7 @@ def initialize(project, deploy_groups)
         @project = project
         @deploy_groups = deploy_groups
         @errors = []
+        @ids = Samson::Secrets::Manager.ids # cache since we use this for every secret
```

Comment:
is this ever not used? worth having a memoized method?

Reply:
always used